SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -45,6 +45,13 @@ enum class ERPCResult : uint8
 	Unknown
 };
 
+enum class ERPCQueueCommand : uint8_t
```

Comment:
NIT - made a bit more sense as this when I was reading, what you think?
```suggestion
enum class ERPCQueueResult : uint8_t
```

Reply:
Hmm, that seems more confusing to me. A ERPCQueueResult to me suggests what is returned after we attempt to queue something. This is used mostly for processing the queue. Maybe ERPCQueueProcessResult.

Reply:
Sure, I think both are more descriptive than `ERPCQueueCommand`